servlet-feature-pack/common/pom.xml

```diff
@@ -299,6 +299,18 @@
             </exclusions>
         </dependency>
 
+        <dependency>
+            <groupId>org.wildfly.core</groupId>
+            <artifactId>wildfly-cli</artifactId>
+            <classifier>client</classifier>
+            <exclusions>
+                <exclusion>
+                    <groupId>*</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
```

Comment:
@ehsavoie @jfdenise I'm a bit concerned this was necessary. Will the 5.2.5 plugin work properly with old feature packs that didn't do this?

If this is just a case of updating the FP to follow current best practices I'm not concerned.

Reply:
@bstansberry the change in 5.2.5 impacts provisioning time. Older feature-packs depend on older versions of the plugin so, during provisioning the older version is used. New WildFly feature-packs that depend on 5.2.5 need this fix.

Reply:
Thanks, Jeff.